tests/features/ecas-register.feature

```diff
@@ -1,4 +1,4 @@
-@javascript @ecas-login
+@javascript @ecas-login @reg
```

Comment:
Remove testing tag and the ecas-login and javascript tags are not needed either.
```suggestion
```